src/apps/loan-list/materials/utils/warning-bar.tsx

```diff
@@ -32,7 +36,7 @@ const WarningBar: FC<WarningBarProps> = ({
             {leftLink && (
               <Link
                 href={leftLink}
-                className="link-tag color-secondary-gray ml-8"
+                className="link-tag color-secondary-gray ml-8 hide-linkstyle"
```

Comment:
Can't we override the link styling in the design system?
We know we have multiple places where a link button is rendered with an underline, and adding this utility class everywhere is not the solution. There should not be a case where we want an underline.

Reply:
@JacobArrow no, we just don't want the text in buttons to be underlined ever.

Reply:
> we just don't want the text in buttons to be underlined ever
But this is a regular link - why remove the underline then?

Reply:
Great point!